### PR TITLE
Feat navigation

### PR DESCRIPTION
**Pull request by hinanshisuthar**
Added Navigation Component
- For desktop 
- For Mobile(Hamburger Menu)

### CODE REVIEW

File: components/navbar.css [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+.dropdown {
+    position: relative;
+    display: inline-block;
+}
+
+.dropdown-content {
+    display: none;
+    position: absolute;
+    background-color: var(--light-grey);
+    min-width: 160px;
```
Comment (omgaikwad) [inline components/navbar.css:66]: can use 'rem' unit here

**Review by omgaikwad — APPROVED** [review]
Code is clean, good work!

File: components/navbar.css [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+    overflow: auto;
+    z-index: 1;
```
Comment (RakshithBhat03) [inline components/navbar.css:68]: If you have a defined variable for `z-index` then use it here.

**Review by RakshithBhat03 — COMMENTED** [review]
The code looks clean and well structured. Good to merge after implementing suggestion.

File: components/navbar.css (hunk 1 above)
Comment (hinanshisuthar) [reply components/navbar.css:66]: I didn't want it to go below beyond this so have used px here.

**Review by ZuberDunge — APPROVED** [review]
Excellent code structure, UI looks pretty too.  Good work Himanshi.

Good to go from my side.

File: components/navbar.css (hunk 2 above)
Comment (hinanshisuthar) [reply components/navbar.css:68]: Yes I have. Would make the change.

File: components/navbar.css (hunk 1 above)
Comment (omgaikwad) [reply components/navbar.css:66]: Cool, you can merge!

**Pull request merged**